invenio_app_ils/records/loaders/schemas/json/document_request.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2018 CERN.
+#
+# invenio-app-ils is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""DocumentRequest schema for marshmallow loader."""
+
+from invenio_records_rest.schemas.fields import PersistentIdentifier
+from marshmallow import Schema, fields
+
+
```

Comment:
A few comments here:
* maybe I am wrong, but it is missing the `metadata` field with inside all these fields, no? to be discussed maybe face to face :)
* I would maybe inherit from `RecordMetadataSchemaJSONV1` and not directly `Schema` so you already get the `pid` field.
* I would change `document_pid = fields.Str()` to `document_pid = fields.Str(dump_only=True)`, WDYT?
* I would add `document = fields.Raw(dump_only=True)`

Can we sanitize these strings somehow? In `invenio-records-rest` we have `SanitizedHTML` and `SanitizedUnicode`, I am not sure it can be useful...